### PR TITLE
Update make-a-bet.md

### PR DESCRIPTION
**Pull request by Jt13-0x**

### CODE REVIEW

File: docs/tutorial-usage/make-a-bet.md [hunk 1]
```diff
@@ -4,32 +4,32 @@ sidebar_position: 3
...
 ## Conditions
-
-This what the Judge should use to determine a winner. It is a string of text that could be anything the Maker feels like betting on. It is recommended that the Maker should make clear terms for thier bet conditions to avoid confusion and cancelled bets. 
+ 
+The conditions is what the Judge should review to determine a winner. It is a string of text which could be anything the Maker feels like creating a bet on. To avoid confusion and cancellations, the Maker should clearly define the terms of their bet.
 
 ## Maker
 
-The person making the bet. The address is autopopulated with the logged in user's wallet, but it could technically be anyone. This person will also be placing thier own money into the bet as a wager.
+The Maker is the person making the bet. The address is autopopulated witht the wallet the user logged in with, but it could technically be anyone. This individual will also be placing their own funds into the wager based on the conditions specified.
```
Comment (theSchein) [inline docs/tutorial-usage/make-a-bet.md]: 'witht' spelled incorrectly

File: docs/tutorial-usage/make-a-bet.md [hunk 2]
```diff
@@ -4,32 +4,32 @@ sidebar_position: 3
...
 
 ## Judge
 
-This is the person that decides who wins the bet. It is important to choose the Judge carefully and make sure this is someone that both parties can trust. The conditions of the bet are technically arbitrary for how the bet is resolved, it is entirely up to the descrition of the Judge. The Judge recieves no financial compensation for thier role, this arrangement is entirely trust based. 
+This is the person who decides the outcome of the bet. It’s important to choose the Judge carefully, ensuring they are someone both parties can trust. The bet’s resolution is entirely up to the Judge’s discretion, as the conditions are technically arbitrary. The Judge does not receive any financial compensation; this arrangement is based entirely on trust.
 
-The Judge may be the Maker or the Taker for simplicity. This arrangement is not recommended for high stakes wagers or bets with argumentative people. 
+For simplicity, the Judge can be either the Maker or the Taker, though this setup is not advisable for high-stakes wagers or when dealing with oposing opinions.
```
Comment (theSchein) [inline docs/tutorial-usage/make-a-bet.md]: i still prefer 'argumentative people' over oposing opinions at the end of this sentance

File: docs/tutorial-usage/make-a-bet.md [hunk 3]
```diff
@@ -4,32 +4,32 @@ sidebar_position: 3
...
 
 ## Wager Amount
 
-This is the amount the each side puts into the bet. For example if the wager is worth $100 then the fully funded contract will have $200 in it. While the wagers are entered in USD, currently the bet contract wagers only accept ETH. 
+This is the amount each side puts into the bet. For example, if the wager is worth $100 then the fully funded contract will have a total of $200 in it. While the wagers are entered in USD, currently Ok Bet's wagers only accepts ETH.
```
Comment (theSchein) [inline docs/tutorial-usage/make-a-bet.md]: Okay Bet's

**Review by theSchein — CHANGES_REQUESTED** [review]
minor nits and you are good to go

File: docs/tutorial-usage/make-a-bet.md (hunk 3 above)
Comment (Jt13-0x) [reply docs/tutorial-usage/make-a-bet.md]: updated

**Review by theSchein — APPROVED** [review]

**Pull request merged**